### PR TITLE
#27: 'solve https://github.com/buildo/labonline/issues/724' (closes #27)

### LINKED ISSUE #27: 'solve https://github.com/buildo/labonline/issues/724'

### PR DESCRIPTION
**Pull request by francescogior**
Issue #27

https://github.com/buildo/labonline/issues/724

### CODE REVIEW

**Comment by giogonzo** [comment]
looks good to me, @FrancescoCioria please double check that 100% and merge if ok

File: src/Panel/PanelMenu.js [hunk 1]
```diff
@@ -60,8 +60,8 @@ export default class PanelMenu extends React.Component {
 
   makeDisabledAction = action => (
     <Tooltip
-      popover={{ content: `Disabled because ${action.disabled.reason}`, position: 'top', anchor: 'center' }}
-      style={{ display: 'inline-block', cursor: 'not-allowed' }}
+      popover={{ content: `${action.disabled.reason}`, position: 'top', anchor: 'start' }}
+      style={{ display: 'inline-block', cursor: 'not-allowed', width: '100%' }}
```
Comment (FrancescoCioria) [inline src/Panel/PanelMenu.js]: isn't `display: 'inline-block' + width: 100%` the same of `display: 'block'` ?

Comment (francescogior) [inline src/Panel/PanelMenu.js]: I thought so. But with display:block the metadata text goes to next row

Comment (giogonzo) [inline src/Panel/PanelMenu.js]: well... not sure, but It should be safe to merge in any case, we'll come back if it brakes something

**Pull request merged**